Core/ServeSync.Application/UseCases/EventManagement/EventManagementMapperProfile.cs

```diff
@@ -38,6 +40,9 @@ public EventManagementMapperProfile()
             .ForMember(dest => dest.Name, opt => opt.MapFrom(src => src.Organization!.Name))
             .ForMember(dest => dest.ImageUrl, opt => opt.MapFrom(src => src.Organization!.ImageUrl));
 
+        CreateMap<EventCollaborationRequest, EventCollaborationRequestDto>();
+
         CreateMap<EventAttendanceInfo, AttendanceInfoGenerateQrCodeBackGroundJobDto>();
+
```

Comment:
Don't forget to config map from 2 dtos that you're about to create (EventOrganizationInfoDto, EventOrganizationInfo)